test/test_get.py

```diff
@@ -196,37 +210,147 @@ def test_download_zip_and_expiration():
         "https://github.com/biocypher/biocypher/raw/main/test/test_CSVs.zip",
         lifetime=7,
     )
-    paths = downloader.download(resource)
+    api = APIRequest(
```

Comment:
Why is this test included in `test_download_zip_and_expiration`? I think the test of API request functionality deserves its own battery of tests. Please move to separate functions.